src/Sputnik.jl

```diff
@@ -3,9 +3,10 @@ module Sputnik
 using InteractBase
 using InteractBulma
 using StatPlots
+plotlyjs()
```

Comment:
Is this a good idea? Now essentially global state that a user might have set gets overwritten when they do ``using Sputnik``. That seems really not ideal?

Reply:
I'm still at the stage of "playing around" with Sputnik, and wanted to try out whether it works with PlotlyJS. I'll see if there's a more elegant way of doing this, maybe I can simply try and respect whatever backend the user was using when loading Sputnik.

Reply:
Reverted: now it respects whatever backend the user was using, that's probably the cleanest way to do it (I need an explicit `import GR` as otherwise I get some world age errors for some reason but that doesnt impact backend choice)